ocs_ci/utility/utils.py

```diff
@@ -352,6 +353,12 @@ def custom_ceph_config(suite_config, custom_config, custom_config_file):
     return full_custom_config
 
 
+def quiet_logs(quiet):
+    if quiet:
+        logging.disable(sys.maxsize)
+    return quiet
```

Comment:
1) no need to return this value

Reply:
The problem is that I need to keep it in case a CommandFailed is raised. By default, it prints out the command that failed. If we'll simply silence it, we won't know which command failed.
Alternatively I could replace .pop with .get, but in that case, `quiet` would be sent over to `subprocess.run` and would crash

Reply:
ack